esp/esp/dbmail/cronmail.py

```diff
@@ -42,6 +42,10 @@
 
 from django.conf import settings
 
+
+_ONE_WEEK = timedelta(weeks=1)
+
+
 @transaction.autocommit
```

Comment:
Yes.  In 1.6+, roughly the same behavior is the default.  (We have middleware to change that, but it doesn't apply here.)